src/main/java/org/terasology/web/resources/ResourceManager.java

```diff
@@ -94,9 +97,11 @@ public void initialize(TerasologyEngine gameEngine) {
                         .addSubResource("MOTD", new ServerMotdResource())
                         .build())
                 .addSubResource("serverAdmins", new ServerAdminsResource())
+                .addSubResource("serverAdminPermissions", new AdminPermissionListResource())
                 .addSubResource("system", systemResource)
                 .build();
         systemResource.startSystemInfoRefreshService();
+        InjectionHelper.inject(AdminPermissionManager.getInstance(), context);
```

Comment:
`@In`s and `InjectionHelper` are mostly useful when you have to inject the values in a lot of classes, to avoid the verbosity of passing them in each constructor and similar situations. However here there is just one class, so I'd have just put something like a `setNetworkSystem` method and called it here (or passed it to the constructor, but not really feasible in this case since it's a singleton). You should be able to obtain the `NetworkSystem` from `context` with a `context.get(NetworkSystem.class)` - which is basically what `InjectionHelper` does IIRC, it looks at the types of the fields to be injected and queries the context for an instance of those classes